src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: number => {
+      calculator.result += number;
+    },
```

Comment:
```suggestion
    add(number) {
      this.result += number;
    },
```
Try to make it work like that everywhere